packages/inter-protocol/src/proposals/price-feed-proposal.js

```diff
@@ -98,13 +97,14 @@ export const createPriceFeed = async (
       chainTimerService,
       client,
       econCharterKit,
-      economicCommitteeCreatorFacet,
       highPrioritySendersManager,
       namesByAddressAdmin,
       priceAuthority,
       priceAuthorityAdmin,
-      zoe,
+      startGovernedUpgradeable: startGovernedUpgradeableP,
+      fluxAggregatorKits,
     },
+    produce: { fluxAggregatorKits: produceFluxAggregatorKits }, // TODO - make a different kit for each asset
```

Comment:
TODO when?

why is there a `fluxAggregatorKits` promise at all?

Reply:
This is actually a stale comment. This is an array that contains each kit. It's just to be safe because they have admin facets, in case they ever need to be upgraded (unlikely).